# **ANALYSE ET TRAITEMENT ALL DATA EHCVM - ANSTAT**

# **MISE EN SITUATION**

# **RECUPERATION DES DONNEES**

In [1]:
#importation des bibliothèques

import pandas as pd
import numpy as np
import seaborn as sns
import matplotlib.pyplot as plt

In [2]:
# Load data

df = pd.read_csv("../DATAS/ALL_EHCVM.csv")

# **DESCRIPTION ET OBJECTIFS**

# **ANALYSE EXPLORATOIRE DES DONNEES**

In [3]:
# Lecture du dataframe

df.head()

idmen  idind              region      sexe              mstat  \
0      2      4  AUTONOME D'ABIDJAN   Féminin        Célibataire   
1      2      5  AUTONOME D'ABIDJAN   Féminin        Célibataire   
2      2      3  AUTONOME D'ABIDJAN  Masculin        Célibataire   
3      2      6  AUTONOME D'ABIDJAN   Féminin        Célibataire   
4      2      2  AUTONOME D'ABIDJAN   Féminin  Marié(e) monogame   

          branch                            sectins  \
0  Aut. services  Entreprise publique/ parapublique   
1            NaN                                NaN   
2            NaN                                NaN   
3            NaN                                NaN   
4            NaN                                NaN   

                             csp  rev_total_mois  log_revenu  sans_revenu  \
0  Cadre moyen/agent de maîtrise        279244.0   12.539845            0   
1                            NaN             0.0    0.000000            1   
2                            NaN             0.0    0.000000            1   
3                            NaN             0.0    0.000000            1   
4                            NaN             0.0    0.000000            1   

  age_grp  age_num  empl_formel  bancarise  a_assurance      logem  
0   25-34       29            1          1            0  Locataire  
1    0-17       17            0          0            0  Locataire  
2    0-17       15            0          0            0  Locataire  
3    0-17       12            0          0            0  Locataire  
4   25-34       34            0          0            0  Locataire

In [4]:
# Copy du dataframe afin d'avoir toujours une version originale

df_ehcvm = df.copy()

In [5]:
df_ehcvm

idmen  idind              region      sexe              mstat  \
0          2      4  AUTONOME D'ABIDJAN   Féminin        Célibataire   
1          2      5  AUTONOME D'ABIDJAN   Féminin        Célibataire   
2          2      3  AUTONOME D'ABIDJAN  Masculin        Célibataire   
3          2      6  AUTONOME D'ABIDJAN   Féminin        Célibataire   
4          2      2  AUTONOME D'ABIDJAN   Féminin  Marié(e) monogame   
...      ...    ...                 ...       ...                ...   
64486  13863  64490             MORONOU  Masculin        Célibataire   
64487  13863  64489             MORONOU  Masculin        Célibataire   
64488  13863  64491             MORONOU  Masculin        Célibataire   
64489  13863  64488             MORONOU  Masculin        Célibataire   
64490  13863  64486             MORONOU  Masculin  Marié(e) monogame   

                   branch                            sectins  \
0           Aut. services  Entreprise publique/ parapublique   
1                     NaN                                NaN   
2                     NaN                                NaN   
3                     NaN                                NaN   
4                     NaN                                NaN   
...                   ...                                ...   
64486                 NaN                                NaN   
64487                 NaN                                NaN   
64488                 NaN                                NaN   
64489                 NaN                                NaN   
64490  Elevage/syl./peche                  Entreprise Privée   

                                  csp  rev_total_mois  log_revenu  \
0       Cadre moyen/agent de maîtrise        279244.0   12.539845   
1                                 NaN             0.0    0.000000   
2                                 NaN             0.0    0.000000   
3                                 NaN             0.0    0.000000   
4                                 NaN             0.0    0.000000   
...                               ...             ...         ...   
64486                             NaN             0.0    0.000000   
64487                             NaN             0.0    0.000000   
64488                             NaN             0.0    0.000000   
64489                             NaN             0.0    0.000000   
64490  Travailleur pour compte propre             0.0    0.000000   

       sans_revenu age_grp  age_num  empl_formel  bancarise  a_assurance  \
0                0   25-34       29            1          1            0   
1                1    0-17       17            0          0            0   
2                1    0-17       15            0          0            0   
3                1    0-17       12            0          0            0   
4                1   25-34       34            0          0            0   
...            ...     ...      ...          ...        ...          ...   
64486            1    0-17       11            0          0            0   
64487            1    0-17        7            0          0            0   
64488            1    0-17       10            0          0            0   
64489            1    0-17        4            0          0            0   
64490            1   35-44       36            0          1            0   

                    logem  
0               Locataire  
1               Locataire  
2               Locataire  
3               Locataire  
4               Locataire  
...                   ...  
64486  Proprietaire titre  
64487  Proprietaire titre  
64488  Proprietaire titre  
64489  Proprietaire titre  
64490  Proprietaire titre  

[64491 rows x 17 columns]

In [6]:
# Nombre lignes et colonnes

df_ehcvm.shape

(64491, 17)

In [7]:
# Infos dataframe

df_ehcvm.info(verbose=True)

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 64491 entries, 0 to 64490
Data columns (total 17 columns):
 #   Column          Non-Null Count  Dtype  
---  ------          --------------  -----  
 0   idmen           64491 non-null  int64  
 1   idind           64491 non-null  int64  
 2   region          64491 non-null  object 
 3   sexe            64491 non-null  object 
 4   mstat           64491 non-null  object 
 5   branch          26294 non-null  object 
 6   sectins         26294 non-null  object 
 7   csp             26294 non-null  object 
 8   rev_total_mois  64491 non-null  float64
 9   log_revenu      64491 non-null  float64
 10  sans_revenu     64491 non-null  int64  
 11  age_grp         64491 non-null  object 
 12  age_num         64491 non-null  int64  
 13  empl_formel     64491 non-null  int64  
 14  bancarise       64491 non-null  int64  
 15  a_assurance     64491 non-null  int64  
 16  logem           64491 non-null  object 
dtypes: float64(2), int64(7), object

In [8]:
# Description du Dataframe

df_ehcvm.describe()

idmen         idind  rev_total_mois    log_revenu   sans_revenu  \
count  64491.000000  64491.000000    6.449100e+04  64491.000000  64491.000000   
mean    7086.913678  32246.000000    8.832082e+03      0.854420      0.924253   
std     3947.746739  18617.092442    4.837005e+04      2.993787      0.264595   
min        2.000000      1.000000    0.000000e+00      0.000000      0.000000   
25%     3727.000000  16123.500000    0.000000e+00      0.000000      1.000000   
50%     7173.000000  32246.000000    0.000000e+00      0.000000      1.000000   
75%    10389.000000  48368.500000    0.000000e+00      0.000000      1.000000   
max    13863.000000  64491.000000    2.522333e+06     14.740695      1.000000   

           age_num   empl_formel     bancarise   a_assurance  
count  64491.00000  64491.000000  64491.000000  64491.000000  
mean      23.28083      0.012219      0.267464      0.034858  
std       19.32949      0.109862      0.442640      0.183420  
min        0.00000      0.000000      0.000000      0.000000  
25%        8.00000      0.000000      0.000000      0.000000  
50%       17.00000      0.000000      0.000000      0.000000  
75%       36.00000      0.000000      1.000000      0.000000  
max      122.00000      1.000000      1.000000      1.000000

In [9]:
# Valeurs manquantes

df_ehcvm.isna().sum()

idmen                 0
idind                 0
region                0
sexe                  0
mstat                 0
branch            38197
sectins           38197
csp               38197
rev_total_mois        0
log_revenu            0
sans_revenu           0
age_grp               0
age_num               0
empl_formel           0
bancarise             0
a_assurance           0
logem                 0
dtype: int64

In [10]:
# Nombre de valeurs uniques

df_ehcvm.nunique()

idmen             12965
idind             64491
region               33
sexe                  2
mstat                 7
branch               10
sectins               8
csp                  13
rev_total_mois     1381
log_revenu         1379
sans_revenu           2
age_grp               7
age_num             106
empl_formel           2
bancarise             2
a_assurance           2
logem                 5
dtype: int64

In [11]:
# Listes des montants salaire uniques

listes = sorted(df_ehcvm["rev_total_mois"].unique().tolist())
df_listes = pd.DataFrame(listes, columns=["rev_total_mois_unique"])

# Exporter en CSV
df_listes.to_csv("revenus_uniques.csv", index=False, encoding="utf-8")

In [12]:
# Individu ayant le plus gros salaire mensuel
df_e = df_ehcvm[df_ehcvm["rev_total_mois"] == df_ehcvm["rev_total_mois"].max()]
df_e

idmen  idind              region      sexe              mstat  \
2431    602   2430  AUTONOME D'ABIDJAN  Masculin  Marié(e) monogame   

               branch                            sectins              csp  \
2431  Education/Sante  Entreprise publique/ parapublique  Cadre supérieur   

      rev_total_mois  log_revenu  sans_revenu age_grp  age_num  empl_formel  \
2431       2522333.3   14.740695            0   55-64       59            1   

      bancarise  a_assurance               logem  
2431          1            0  Proprietaire titre

In [13]:
# Listing des colonnes du dataframe

df_ehcvm.columns

Index(['idmen', 'idind', 'region', 'sexe', 'mstat', 'branch', 'sectins', 'csp',
       'rev_total_mois', 'log_revenu', 'sans_revenu', 'age_grp', 'age_num',
       'empl_formel', 'bancarise', 'a_assurance', 'logem'],
      dtype='object')

# **NETTOYAGE DES DONNEES**

## **Traitement des valeurs manquantes** 

In [14]:
val = "Non_connu"

df_ehcvm["branch"] = df_ehcvm["branch"].fillna(val)
df_ehcvm["sectins"] = df_ehcvm["sectins"].fillna(val)
df_ehcvm["csp"] = df_ehcvm["csp"].fillna(val)

In [15]:
df_ehcvm.isna().sum()

idmen             0
idind             0
region            0
sexe              0
mstat             0
branch            0
sectins           0
csp               0
rev_total_mois    0
log_revenu        0
sans_revenu       0
age_grp           0
age_num           0
empl_formel       0
bancarise         0
a_assurance       0
logem             0
dtype: int64

In [16]:
df_ehcvm

idmen  idind              region      sexe              mstat  \
0          2      4  AUTONOME D'ABIDJAN   Féminin        Célibataire   
1          2      5  AUTONOME D'ABIDJAN   Féminin        Célibataire   
2          2      3  AUTONOME D'ABIDJAN  Masculin        Célibataire   
3          2      6  AUTONOME D'ABIDJAN   Féminin        Célibataire   
4          2      2  AUTONOME D'ABIDJAN   Féminin  Marié(e) monogame   
...      ...    ...                 ...       ...                ...   
64486  13863  64490             MORONOU  Masculin        Célibataire   
64487  13863  64489             MORONOU  Masculin        Célibataire   
64488  13863  64491             MORONOU  Masculin        Célibataire   
64489  13863  64488             MORONOU  Masculin        Célibataire   
64490  13863  64486             MORONOU  Masculin  Marié(e) monogame   

                   branch                            sectins  \
0           Aut. services  Entreprise publique/ parapublique   
1               Non_connu                          Non_connu   
2               Non_connu                          Non_connu   
3               Non_connu                          Non_connu   
4               Non_connu                          Non_connu   
...                   ...                                ...   
64486           Non_connu                          Non_connu   
64487           Non_connu                          Non_connu   
64488           Non_connu                          Non_connu   
64489           Non_connu                          Non_connu   
64490  Elevage/syl./peche                  Entreprise Privée   

                                  csp  rev_total_mois  log_revenu  \
0       Cadre moyen/agent de maîtrise        279244.0   12.539845   
1                           Non_connu             0.0    0.000000   
2                           Non_connu             0.0    0.000000   
3                           Non_connu             0.0    0.000000   
4                           Non_connu             0.0    0.000000   
...                               ...             ...         ...   
64486                       Non_connu             0.0    0.000000   
64487                       Non_connu             0.0    0.000000   
64488                       Non_connu             0.0    0.000000   
64489                       Non_connu             0.0    0.000000   
64490  Travailleur pour compte propre             0.0    0.000000   

       sans_revenu age_grp  age_num  empl_formel  bancarise  a_assurance  \
0                0   25-34       29            1          1            0   
1                1    0-17       17            0          0            0   
2                1    0-17       15            0          0            0   
3                1    0-17       12            0          0            0   
4                1   25-34       34            0          0            0   
...            ...     ...      ...          ...        ...          ...   
64486            1    0-17       11            0          0            0   
64487            1    0-17        7            0          0            0   
64488            1    0-17       10            0          0            0   
64489            1    0-17        4            0          0            0   
64490            1   35-44       36            0          1            0   

                    logem  
0               Locataire  
1               Locataire  
2               Locataire  
3               Locataire  
4               Locataire  
...                   ...  
64486  Proprietaire titre  
64487  Proprietaire titre  
64488  Proprietaire titre  
64489  Proprietaire titre  
64490  Proprietaire titre  

[64491 rows x 17 columns]

## **Transforme variable marié** 

In [17]:
val_mar = "Marié(e)"

df_ehcvm["mstat"] = df_ehcvm["mstat"].replace(["Marié(e) polygame", "Marié(e) monogame"], val_mar)

In [18]:
df_ehcvm

idmen  idind              region      sexe        mstat  \
0          2      4  AUTONOME D'ABIDJAN   Féminin  Célibataire   
1          2      5  AUTONOME D'ABIDJAN   Féminin  Célibataire   
2          2      3  AUTONOME D'ABIDJAN  Masculin  Célibataire   
3          2      6  AUTONOME D'ABIDJAN   Féminin  Célibataire   
4          2      2  AUTONOME D'ABIDJAN   Féminin     Marié(e)   
...      ...    ...                 ...       ...          ...   
64486  13863  64490             MORONOU  Masculin  Célibataire   
64487  13863  64489             MORONOU  Masculin  Célibataire   
64488  13863  64491             MORONOU  Masculin  Célibataire   
64489  13863  64488             MORONOU  Masculin  Célibataire   
64490  13863  64486             MORONOU  Masculin     Marié(e)   

                   branch                            sectins  \
0           Aut. services  Entreprise publique/ parapublique   
1               Non_connu                          Non_connu   
2               Non_connu                          Non_connu   
3               Non_connu                          Non_connu   
4               Non_connu                          Non_connu   
...                   ...                                ...   
64486           Non_connu                          Non_connu   
64487           Non_connu                          Non_connu   
64488           Non_connu                          Non_connu   
64489           Non_connu                          Non_connu   
64490  Elevage/syl./peche                  Entreprise Privée   

                                  csp  rev_total_mois  log_revenu  \
0       Cadre moyen/agent de maîtrise        279244.0   12.539845   
1                           Non_connu             0.0    0.000000   
2                           Non_connu             0.0    0.000000   
3                           Non_connu             0.0    0.000000   
4                           Non_connu             0.0    0.000000   
...                               ...             ...         ...   
64486                       Non_connu             0.0    0.000000   
64487                       Non_connu             0.0    0.000000   
64488                       Non_connu             0.0    0.000000   
64489                       Non_connu             0.0    0.000000   
64490  Travailleur pour compte propre             0.0    0.000000   

       sans_revenu age_grp  age_num  empl_formel  bancarise  a_assurance  \
0                0   25-34       29            1          1            0   
1                1    0-17       17            0          0            0   
2                1    0-17       15            0          0            0   
3                1    0-17       12            0          0            0   
4                1   25-34       34            0          0            0   
...            ...     ...      ...          ...        ...          ...   
64486            1    0-17       11            0          0            0   
64487            1    0-17        7            0          0            0   
64488            1    0-17       10            0          0            0   
64489            1    0-17        4            0          0            0   
64490            1   35-44       36            0          1            0   

                    logem  
0               Locataire  
1               Locataire  
2               Locataire  
3               Locataire  
4               Locataire  
...                   ...  
64486  Proprietaire titre  
64487  Proprietaire titre  
64488  Proprietaire titre  
64489  Proprietaire titre  
64490  Proprietaire titre  

[64491 rows x 17 columns]

## **Nettoyage variable logement** 

In [19]:
df_ehcvm["logem"].unique()

array(['Locataire', 'Autre', 'Proprietaire titre',
       'Proprietaire sans titre', '9'], dtype=object)

In [20]:
df_ehc = df_ehcvm[df_ehcvm["logem"] == "9"]
df_ehc

idmen  idind              region      sexe        mstat  \
755      216    760  AUTONOME D'ABIDJAN   Féminin  Célibataire   
756      216    756  AUTONOME D'ABIDJAN  Masculin     Marié(e)   
757      216    758  AUTONOME D'ABIDJAN  Masculin  Célibataire   
758      216    759  AUTONOME D'ABIDJAN  Masculin  Célibataire   
759      216    757  AUTONOME D'ABIDJAN   Féminin     Marié(e)   
8739    1998   8740                PORO  Masculin     Marié(e)   
8740    1998   8741                PORO   Féminin     Marié(e)   
8765    2004   8766                PORO   Féminin     Veuf(ve)   
8766    2004   8767                PORO   Féminin  Célibataire   
8767    2004   8768                PORO   Féminin  Célibataire   
31985   7124  31986         LÔH-DJIBOUA   Féminin  Célibataire   
57852  12477  57855               LA ME   Féminin  Célibataire   
57853  12477  57854               LA ME   Féminin  Célibataire   
57854  12477  57856               LA ME   Féminin  Célibataire   
57855  12477  57858               LA ME   Féminin  Célibataire   
57856  12477  57853               LA ME   Féminin     Veuf(ve)   
57857  12477  57857               LA ME  Masculin  Célibataire   

                branch                                         sectins  \
755          Non_connu                                       Non_connu   
756       Trans./Comm.                               Entreprise Privée   
757          Non_connu                                       Non_connu   
758          Non_connu                                       Non_connu   
759      Autr. indust.                               Entreprise Privée   
8739     Aut. services                               Entreprise Privée   
8740         Non_connu                                       Non_connu   
8765   Education/Sante                               Entreprise Privée   
8766         Non_connu                                       Non_connu   
8767         Non_connu                                       Non_connu   
31985  Education/Sante                      Etat/Collectivités locales   
57852        Non_connu                                       Non_connu   
57853        Non_connu                                       Non_connu   
57854        Non_connu                                       Non_connu   
57855    Aut. services  Ménage comme employeur de personnel domestique   
57856  Education/Sante                      Etat/Collectivités locales   
57857    Autr. indust.                               Entreprise Privée   

                                  csp  rev_total_mois  log_revenu  \
755                         Non_connu             0.0    0.000000   
756                   Cadre supérieur        300000.0   12.611541   
757                         Non_connu             0.0    0.000000   
758                         Non_connu             0.0    0.000000   
759    Travailleur pour compte propre             0.0    0.000000   
8739                 Patron/Employeur             0.0    0.000000   
8740                        Non_connu             0.0    0.000000   
8765      Ouvrier ou employé qualifié        100000.0   11.512936   
8766                        Non_connu             0.0    0.000000   
8767                        Non_connu             0.0    0.000000   
31985     Ouvrier ou employé qualifié        245000.0   12.409018   
57852                       Non_connu             0.0    0.000000   
57853                       Non_connu             0.0    0.000000   
57854                       Non_connu             0.0    0.000000   
57855         Manœuvre, aide ménagère         15000.0    9.615872   
57856   Cadre moyen/agent de maîtrise        443000.0   13.001328   
57857  Travailleur pour compte propre             0.0    0.000000   

       sans_revenu age_grp  age_num  empl_formel  bancarise  a_assurance logem  
755              1    0-17        8            0          0            0     9  
756              0   45-54       54            1          1            0     9  

In [21]:
df_ehc.count()

idmen             17
idind             17
region            17
sexe              17
mstat             17
branch            17
sectins           17
csp               17
rev_total_mois    17
log_revenu        17
sans_revenu       17
age_grp           17
age_num           17
empl_formel       17
bancarise         17
a_assurance       17
logem             17
dtype: int64

In [22]:
df_ehcvm.count()

idmen             64491
idind             64491
region            64491
sexe              64491
mstat             64491
branch            64491
sectins           64491
csp               64491
rev_total_mois    64491
log_revenu        64491
sans_revenu       64491
age_grp           64491
age_num           64491
empl_formel       64491
bancarise         64491
a_assurance       64491
logem             64491
dtype: int64

#### 17 par rapport à 64491 est négligeable.
### 64491 -> 100%
### 17 -> ?%
### 17 -> 0.03% des données
#### Ce retrouver donc avec 64474 / 64491 données cleans, fiables est acceptable.
#### Donc supprimons les lignes manquantes

In [23]:
df_ehcvm = df_ehcvm.drop(df_ehc.index)
df_ehcvm.count()

idmen             64474
idind             64474
region            64474
sexe              64474
mstat             64474
branch            64474
sectins           64474
csp               64474
rev_total_mois    64474
log_revenu        64474
sans_revenu       64474
age_grp           64474
age_num           64474
empl_formel       64474
bancarise         64474
a_assurance       64474
logem             64474
dtype: int64

## **Variables pour les Statistiques population** 

In [24]:
df_stat = df_ehcvm.drop(['idmen', 'idind', 'log_revenu', 'sans_revenu'], axis=1)
df_stat

region      sexe        mstat              branch  \
0      AUTONOME D'ABIDJAN   Féminin  Célibataire       Aut. services   
1      AUTONOME D'ABIDJAN   Féminin  Célibataire           Non_connu   
2      AUTONOME D'ABIDJAN  Masculin  Célibataire           Non_connu   
3      AUTONOME D'ABIDJAN   Féminin  Célibataire           Non_connu   
4      AUTONOME D'ABIDJAN   Féminin     Marié(e)           Non_connu   
...                   ...       ...          ...                 ...   
64486             MORONOU  Masculin  Célibataire           Non_connu   
64487             MORONOU  Masculin  Célibataire           Non_connu   
64488             MORONOU  Masculin  Célibataire           Non_connu   
64489             MORONOU  Masculin  Célibataire           Non_connu   
64490             MORONOU  Masculin     Marié(e)  Elevage/syl./peche   

                                 sectins                             csp  \
0      Entreprise publique/ parapublique   Cadre moyen/agent de maîtrise   
1                              Non_connu                       Non_connu   
2                              Non_connu                       Non_connu   
3                              Non_connu                       Non_connu   
4                              Non_connu                       Non_connu   
...                                  ...                             ...   
64486                          Non_connu                       Non_connu   
64487                          Non_connu                       Non_connu   
64488                          Non_connu                       Non_connu   
64489                          Non_connu                       Non_connu   
64490                  Entreprise Privée  Travailleur pour compte propre   

       rev_total_mois age_grp  age_num  empl_formel  bancarise  a_assurance  \
0            279244.0   25-34       29            1          1            0   
1                 0.0    0-17       17            0          0            0   
2                 0.0    0-17       15            0          0            0   
3                 0.0    0-17       12            0          0            0   
4                 0.0   25-34       34            0          0            0   
...               ...     ...      ...          ...        ...          ...   
64486             0.0    0-17       11            0          0            0   
64487             0.0    0-17        7            0          0            0   
64488             0.0    0-17       10            0          0            0   
64489             0.0    0-17        4            0          0            0   
64490             0.0   35-44       36            0          1            0   

                    logem  
0               Locataire  
1               Locataire  
2               Locataire  
3               Locataire  
4               Locataire  
...                   ...  
64486  Proprietaire titre  
64487  Proprietaire titre  
64488  Proprietaire titre  
64489  Proprietaire titre  
64490  Proprietaire titre  

[64474 rows x 13 columns]

## **Variables caractéristiques pour modèle de score de solvabilité** 

In [25]:
# Variables caractéristiques pour modèle de score de solvabilité

df_ml = df_ehcvm.drop(['idmen', 'idind', 'region', 'sexe', 'branch', 'sectins', 'csp', 'age_num', 'log_revenu', 'sans_revenu'], axis=1)
df_ml

mstat  rev_total_mois age_grp  empl_formel  bancarise  \
0      Célibataire        279244.0   25-34            1          1   
1      Célibataire             0.0    0-17            0          0   
2      Célibataire             0.0    0-17            0          0   
3      Célibataire             0.0    0-17            0          0   
4         Marié(e)             0.0   25-34            0          0   
...            ...             ...     ...          ...        ...   
64486  Célibataire             0.0    0-17            0          0   
64487  Célibataire             0.0    0-17            0          0   
64488  Célibataire             0.0    0-17            0          0   
64489  Célibataire             0.0    0-17            0          0   
64490     Marié(e)             0.0   35-44            0          1   

       a_assurance               logem  
0                0           Locataire  
1                0           Locataire  
2                0           Locataire  
3                0           Locataire  
4                0           Locataire  
...            ...                 ...  
64486            0  Proprietaire titre  
64487            0  Proprietaire titre  
64488            0  Proprietaire titre  
64489            0  Proprietaire titre  
64490            0  Proprietaire titre  

[64474 rows x 7 columns]

In [26]:
df_ml.columns

Index(['mstat', 'rev_total_mois', 'age_grp', 'empl_formel', 'bancarise',
       'a_assurance', 'logem'],
      dtype='object')

## **Exportation Clean_Datas**

In [27]:
# Pour le modèle

df_ml.to_csv("../DATAS/CleanALL_EHCVM_ML.csv", index=False, encoding="utf-8")

In [28]:
# Pour les statistiques

df_stat.to_csv("../DATAS/CleanALL_EHCVM.csv", index=False, encoding="utf-8")

# **RESUME**